server/service/src/invoice/validate.rs

```diff
@@ -10,6 +10,13 @@ pub fn check_invoice_type(invoice: &InvoiceRow, r#type: InvoiceType) -> bool {
     false
 }
 
+pub fn check_invoice_is_linked(invoice: &InvoiceRow) -> bool {
+    if invoice.linked_invoice_id.is_none() {
+        return true;
+    }
+    false
```

Comment:
`is_none` is already returning a boolean 🤷‍♀️ 
```suggestion
    invoice.linked_invoice_id.is_none()
```

Though actually! This function should be called `check_invoice_is_not_linked`!

Reply:
Have updated this!